easybuild/framework/easyblock.py

```diff
@@ -1444,8 +1444,11 @@ def extensions_step(self, fetch=False):
         self.clean_up_fake_module(fake_mod_data)
 
     def package_step(self):
-        """Package software (e.g. into an RPM)."""
-        pass
+        """Prepare package software (e.g. into an RPM) with fpm."""
+
```

Comment:
I share concerns of @stdweird (and think about it occasionally when building for docker, etc). Do we know what the license is in EB? I don't mind including some advice to the user when packaging things that are restricted.